resource-managers/kubernetes/core/src/main/scala/org/apache/spark/shuffle/KubernetesLocalDiskShuffleExecutorComponents.scala

```diff
@@ -98,8 +98,11 @@ object KubernetesLocalDiskShuffleExecutorComponents extends Logging {
       logInfo(s"Try to recover ${f.getAbsolutePath}")
       try {
         val id = BlockId(f.getName)
-        val decryptedSize = f.length()
-        bm.TempFileBasedBlockStoreUpdater(id, level, classTag, f, decryptedSize).save()
+        // To make it sure to handle only shuffle blocks
+        if (id.isShuffle) {
+          val decryptedSize = f.length()
+          bm.TempFileBasedBlockStoreUpdater(id, level, classTag, f, decryptedSize).save()
+        }
```

Comment:
Do we need to log it? For example we did log for `UnrecognizedBlockId` below.

Reply:
Thank you for review. Sure, let me add a log with `else` statement.